### ISSUE TITLE
OTB cheating

### ISSUE THREAD

**Issue by anuchandra** [clarkerubber/irwin#22]
In view of the latest cheating allegations, I was looking into testing GM OTB games. I could probably get a database of pgn games for training. Just wondering about lichess API integration - is it possible to run this repo in standalone mode without integrating with lichess?

**Comment by styborski1** [comment]
Hey anuchandra, you can definitely run Irwin standalone with some (a lot?) modifications to remove Lichess - the caveat being Lichess won't give you any training data nor will they help you with the configuration. Lichess is open source, but that doesn't mean they will help you understand some of their anti-cheat protections. I've used some of their code in the past, even pulling some Irwin code before for my own use, but I'm just giving you my experience with them from discord, etc. The code is there, but don't expect help on this front. (I'm not at all affiliated with Lichess, just a casual observer/programmer)

**Comment by scchess** [comment]
True. Also The Irwin project is part of the main code base. This repo is old.
